types/sticky-session/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for sticky-session 1.1
+// Project: https://github.com/indutny/sticky-session
+// Definitions by: Giovanni Barbaro <https://github.com/vanny96>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Server } from "http";
+import { ProcessEnvOptions } from "child_process";
```

Comment:
```suggestion
```

You just need to remove the `ProcessEnvOptions` which is probably left by accident.

Reply:
Yes I left them when I was playing around with the `env` type, I then opted for `any` instead but forgot to remove the import! I have updated the pull request! Let me know if there is anything more I have to change!